components/03-modules/slider/_slider.scss

```diff
@@ -63,83 +44,6 @@ $slider__icon-fill-hover           : darken($color-primary, 10%) !default;
         }
     }
 
-    &__link {
-        text-decoration: none;
-
-        &:hover {
-            text-decoration: none;
-        }
-    }
-
-    &__image {
-        width: 100%;
-        margin: $slider__image-margin;
-    }
-
-    &__item-handler {
-        display: flex;
-        flex-wrap: wrap;
-
-        @include mq($screen-m) {
-            flex-wrap: nowrap;
-            justify-content: space-between;
-        }
-    }
-
-    &__name {
-        flex-basis: 100%;
-        font-weight: $slider__name-font-weight;
-
-        @include mq($screen-m) {
-            flex-basis: auto;
-            margin-right: $slider__name-margin-right;
-            margin-bottom: 0;
-        }
-
-        @include mq($screen-l) {
-            font-size: $slider__name-font-size;
-        }
-    }
-
-    &__price {
-        display: flex;
-        flex-wrap: wrap;
-        align-items: center;
-        color: $slider__price-color;
-        font-weight: $slider__price-font-weight;
-
-        @include mq($screen-m) {
-            flex-flow: column;
-            align-items: flex-end;
-            white-space: nowrap;
-        }
-    }
-
-    &__price-old {
-        display: block;
-        margin-right: $slider__price-old-margin;
-        color: $slider__price-color;
-        font-weight: $slider__price-old-font-weight;
-        font-size: $slider__price-old-font-size;
-        text-decoration: line-through;
-
-        @include mq($screen-m) {
-            margin-right: 0;
-        }
-    }
-
-    &__price-new {
-        color: $slider__price-new-color;
-        font-size: $slider__price-new-font-size;
-        font-weight: $slider__price-new-font-weight;
-    }
-
-    &__badge {
-        position: absolute;
-        top: $slider__badge-top;
-        left: 0;
-    }
-
     &__navigation {
```

Comment:
It's a problem with `slider__item` height. On FF and Safari it seems ok but on Chrome the width is much bigger than the image (I put the image from banner and test different configuration). The `height: fit-content` or `height: max-content` for  `slider__item` fix the issue but it not [fully supported](https://caniuse.com/#search=fit-content) so it must be tested on mobiles.

Reply:
error by type of variables (fixed for now) or error by lazyloading images and no placeholder for it